app/assets/javascripts/components/form/form_body.es6.jsx

```diff
@@ -8,13 +8,26 @@ class FormBody extends Component {
     return {};
   }
 
+//TODO(Sonia): Add more styling to container
   get styles() {
-    return {};
-  }
+    return {
+      container: Object.assign( 
+        {},
+        StyleConstants.cards.default,
+        {
+          margin: '20px',
+        }
+      ),
+      ),
+      title: {
+        fontSize: StyleConstants.fonts.sizes.largest,
+      }
+    };
 
   render() {
     return(
-      <div>
+      <div style={this.styles.container} >
```

Comment:
No need for space at the end of this div declaration: `<div style={this.styles.container}>`.